api/src/Entity/CampCollaboration.php

```diff
@@ -202,9 +202,25 @@ class CampCollaboration extends BaseEntity implements BelongsToCampInterface {
     #[ORM\Column(type: 'string', length: 16, nullable: false)]
     public string $role;
 
-    #[ApiProperty(readable: false, writable: false)]
+    /**
+     * The color of the avatar as a hex color string.
+     */
+    #[InputFilter\Trim]
+    #[Assert\Regex(pattern: '/^#[0-9a-zA-Z]{6}$/')]
+    #[ApiProperty(example: '#4DBB52')]
+    #[Groups(['read', 'write'])]
+    #[ORM\Column(type: 'string', length: 8, nullable: true)]
+    public ?string $color = null;
+
+    /**
+     * The abbreviation in the avatar.
+     */
+    #[InputFilter\Trim]
+    #[Assert\Length(max: 2, countUnit: Assert\Length::COUNT_GRAPHEMES)]
```

Comment:
This will allow two emoji, correct? Not that it's a huge issue, but wouldn't COUNT_CODEPOINTS be more correct for these specific rules which we are enforcing? (1 emoji or 2 characters) https://symfony.com/doc/current/reference/constraints/Length.html#countunit

Reply:
Nope, an emoji could be made out of more than two codepoints, that's the reason we use graphemes. But because we want to allow 2 letters as well, we need to allow 2.